nslsii/__init__.py

```diff
@@ -471,3 +475,40 @@ def migrate_metadata():
     os.makedirs(directory, exist_ok=True)
     new_md = PersistentDict(directory)
     new_md.update(old_md)
+
+
+def subscribe_kafka_publisher(RE, beamline_name, bootstrap_servers):
```

Comment:
Should we hard-code a default set of `bootstrap_servers`? If we don't, we'll have to hard-code them in every beamline profile, which will make them harder to change.

Reply:
I think this would be a reasonable place to list the bootstrap servers. There is still a question in my mind about running a kafka broker at each beamline to cache messages in the event connectivity to the kafka cluster is lost.
Either way we could list the CMB machines right now, but does that make sense?

Reply:
I added CMB01, CMB02, and CMB03 to the bootstrap server list.